package.json

```diff
@@ -27,10 +27,12 @@
     "@mui/icons-material": "^5.10.16",
     "@mui/material": "^5.10.17",
     "clsx": "^1.2.1",
+    "docusaurus-plugin-sass": "^0.2.6",
```

Comment:
Are we sure, this is also compatible with the next docusaurus version? Just want to avaoid the same situation we had with the open api plug-in ;)

Reply:
??? Even if it's working now, doesn't mean its supporter for the next docusaurus version